checks.d/unbound.py

```diff
@@ -0,0 +1,106 @@
+import subprocess
+
+from checks import AgentCheck
+
+
+class UnboundCheck(AgentCheck):
+    SERVICE_CHECK_NAME = 'unbound'
+
+
+    def get_stats(self):
+        cmd = 'sudo unbound-control stats'
+        try:
+            output = subprocess.check_output(cmd, stderr=subprocess.STDOUT, shell=True)
+        except subprocess.CalledProcessError as e:
+            error_msg = 'ERROR CALLING {0}: {1}  {2}'.format(cmd, e, e.output)
+            self.service_check(
+                self.SERVICE_CHECK_NAME,
+                AgentCheck.CRITICAL,
+                message=error_msg,
+            )
+            self.log.error()
+            return None
+
+        return output
+
+    def parse_stat(self, stat):
+        label, stat = stat.split("=")
+        prefix, suffix = label.split(".", 1)
+
+        by_tag_labels = [
+            'num.query.flags',
+            'num.query.edns',
+            'num.answer.rcode',
+        ]
+
+        tags = []
+        if prefix.startswith('thread'):
```

Comment:
Thanks Cory! I changed the total metrics to be prefixed with total like it is in the file.  Is there a way to de-pollute datadog of the metrics I've emitted so far?